common/metrics/interfaces.go

```diff
@@ -64,6 +64,9 @@ type (
 		// Scope return an internal scope that can be used to add additional
```

Comment:
as long as your in the file, maybe fix this typo?  
return => returns

there are a few other examples of this typo, including your new code below.

Reply:
Thank you for highlighting this.